.yamllint

```diff
@@ -0,0 +1,29 @@
+---
+extends: default
+ignore: |
+  .github
+  meta*
+  roles/elasticsearch/
```

Comment:
that line is probably copy/pase left over ;)

Reply:
fixed